src/controllers/loginController.ts

```diff
@@ -29,7 +30,7 @@ const loginController = async (req: Request, res: Response) => {
         template: 'email',
         context: {
           name,
-          verificationLink: `${process.env.ROOT_URL}/api/users/verify-email/${verificationToken}`,
+          verificationLink: `${process.env.SWAGGER_ROOT_URL}api/users/verify-email/${verificationToken}`,
```

Comment:
why suddenly change the env variables

Reply:
that was honore's codes we all changed it so verify can work